pkg/controllers/vpcnetworkconfig/cache.go

```diff
@@ -0,0 +1,80 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package vpcnetworkconfig
```

Comment:
There is no store.go for this controller, how about other controller needs to access the cache?

Reply:
please hold off review,  it's possible to move this PR to vpc_dev branch merge.